server/views/routes/emails.js

```diff
@@ -0,0 +1,20 @@
+const router = require('express').Router();
+const User = require('../../models/user');
+const responseHandler = require('../../utils/response-handler');
+
+// Get check if email is alredy created
+router.get('/:email', async (req, res) => {
+  try {
+    const { email } = req.params;
+    const userEmail = await User.findByEmail(email);
+    if (userEmail) {
+      responseHandler.handleSuccess(res, 200, { emailFound: true });
```

Comment:
This if/else block could be simplified to `responseHandler.handleSuccess(res, 200, { emailFound: !!userEmail });

Not critical. Refactor in future PR